seagull.docker

```diff
@@ -0,0 +1,22 @@
+FROM alpine:3.3
+MAINTAINER tobe "tobeg3oogle@gmail.com"
+MAINTAINER Florin Patan "florinpatan@gmail.com"
+
+ARG BINARY_FILE
+
+# DNS stuff
+RUN echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf
+
+# SSL certs
+RUN apk add --update ca-certificates \
```

Comment:
We can skip it for now. This would be useful to allow for ssl connections to binary, so maybe it can be added back when say a self-signed certificate or a user generated certificate will be allowed. I'll update accordingly.